Admin/Pool.php

```diff
@@ -20,22 +20,49 @@
  */
 class Pool
 {
+    /**
+     * @var ContainerInterface|null
+     */
     protected $container = null;
 
+    /**
+     * @var array
+     */
     protected $adminServiceIds = array();
 
+    /**
+     * @var array
+     */
     protected $adminGroups = array();
 
+    /**
+     * @var array
+     */
     protected $adminClasses = array();
 
-    protected $templates    = array();
+    /**
+     * @var array
+     */
+    protected $templates = array();
 
-    protected $assets       = array();
+    /**
+     * @var array
```

Comment:
And removed here: https://github.com/sonata-project/SonataAdminBundle/commit/b4858a41f4e10507485359504720da8f9b6d8619#diff-ed85ead40da3b862ecc9ba97a2e62ac2 so you can remove the property.